### PR TITLE
Fix low power notification never triggering

### LINKED ISSUE #20988: Playtest 20230801: Low Power Audio Alert missing
## Issue Summary

DESCRIPTION
In Playtest 20230801 Red Alert - Single Player Skirmish, Conquest, despite being negative in power, no "Low Power" audio is triggered. This could be tied to #20869.

## Reproduction

STEPS TO REPRODUCE THE ISSUE
1. Start a game of Red Alert, single player, skirmish.
2. Build items to create a deficit of power.

## Expected behavior

EXPECTED BEHAVIOR
When you go below 0 power, an audio alert of "Low Power" should be triggered, or at a minimum, an option to turn this on or off should be included in the Audio menu (like Cash Ticks).

### PR DESCRIPTION
**Pull request by PunkPun**
Closes #20988
Regression from #20869

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public class PowerManager : INotifyCreated, ITick, ISync, IResolveOrder
 
 		public int PowerOutageRemainingTicks { get; private set; }
 		public int PowerOutageTotalTicks { get; private set; }
-		public bool PlayLowPowerNotification { get; set; }
+		public bool PlayLowPowerNotification { get; set; } = true;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs]: Maybe `AdviceInterval > 0` to make it configurable?

Comment (PunkPun) [reply OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs]: makes sense

**Review by Mailaender — APPROVED** [review]

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/8ca4af6cb55988091ee59edb487f385cff638cc5...ab652211b40c4a890a63c0f04952d7cb183c30ba)

**Comment by Mailaender** [comment]
5ce559c853679fdbe61c82aa0edd104b1f79abb1

**Pull request merged**